### ISSUE TITLE
maxResults={n}, paginate={false}, displays all options

### ISSUE THREAD

**Issue by NickBrooks** [ericgio/react-bootstrap-typeahead#208]
I'd like to set my `<Typeahead / >` to show only 5 matching results with no pagination... if I set `maxResults` and set `pagination={false}`, it reverts back to displaying all results... Is this a bug or intended??

Here is my full typeahead code:

```javascript
<Typeahead
             className="form-control editor-hashtags mousetrap"
             ref={ref => this._hashtagTypeahead = ref}
             filterBy={filterHashtagsByCallback}
             multiple
             dropup
             maxResults={5}
             paginate={false}
             options={options}
             placeholder="Hashtags"
             emptyLabel=""
             selected={noteEditor.hashtags}
             onChange={(e) => this.handleHashtagsChange(e)}
             submitFormOnEnter={true}
             />
```

**Comment by ericgio** [comment]
Yeah, that's a bug, thanks.

**Comment by ericgio** [comment]
Fixed in v1.4.2

**Comment by NickBrooks** [comment]
Tested and works as intended. Thanks so much!
